docs/apache-airflow/faq.rst

```diff
@@ -522,3 +522,27 @@ This means ``explicit_defaults_for_timestamp`` is disabled in your mysql server
 
 #. Set ``explicit_defaults_for_timestamp = 1`` under the ``mysqld`` section in your ``my.cnf`` file.
 #. Restart the Mysql server.
+
+Does Airflow collect any telemetry data?
+----------------------------------------
+
+.. _airflow-telemetry-faq:
+
+Airflow integrates `Scarf <https://about.scarf.sh/>`__ to collect basic telemetry data during operation.
+This data assists Airflow maintainers in better understanding how Airflow is used.
+Insights gained from this telemetry are critical for prioritizing patches, minor releases, and
+security fixes. Additionally, this information supports key decisions related to the development road map.
+
+Deployments can opt-out of analytics by setting the :ref:`[telemetry_collection] enabled <config:telemetry_collection__enabled>`
+option to ``False``, or the ``SCARF_ANALYTICS=false`` environment variable.
```

Comment:
+1 here:
```suggestion
option to ``False``, or the ``AIRFLOW__TELEMETRY_COLLECTION__ENABLED=False`` environment variable.
```

Reply:
This is the alternate option. I want to respect what Scarf has in their docs to disable too

https://github.com/apache/airflow/blob/ab1855a8d3a367f5ce36bf25a72ffebfe31448f7/airflow/settings.py#L579-L583

Reply:
If you look at rendered docs it says you can disable it via the "Airflow config" or the Scarf env variable.

The Airflow config can be set via Env Var, airflow.cfg or a Custom Secrets backend

Reply:
Ah, sorry, over-looked this line. Thanks for clarification.